tests/test_components/test_autograd.py

```diff
@@ -610,7 +623,7 @@ def objective(*args):
     assert anp.all(grad != 0.0), "some gradients are 0"
 
     # numerical gradients
-    delta = 1e-1
+    delta = 1e-3
```

Comment:
Clever workaround - not sure I fully understand the tests, should I believe that gradients for all supported Cylinder parameters are tested vs. numerical differences?

Reply:
what's the workaround?

the numerical gradient tests are more things that are done individually to test various combinations of structure (medium / geometry) and monitor. In this case I tested the cylinder parameters and they matched numerical yes.

Reply:
Oh sorry I just meant the general workaround taken in this PR to discretize the cylinder as a polygon. But come to think of it there's no other way, any handling would effectively be doing that.